lib/widgets/string_constants.dart

```diff
@@ -5,3 +5,5 @@ const checkZero = '0';
 const checkMinus = '-';
 const vsLastMonthText = '  vs. last month';
 const balanceAmountText = 'Balance Amount';
+const noDateSelected = 'No date Selected';
+const payDate = 'Pay Date';
```

Comment:
rename to `payDateText`, otherwise, it creates confusion if it is a text or a date.

Reply:
Done